plugins/org.jkiss.dbeaver.model.sql/src/org/jkiss/dbeaver/model/sql/format/SQLFormatUtils.java

```diff
@@ -30,11 +30,21 @@ public class SQLFormatUtils {
     public static String formatSQL(DBPDataSource dataSource, String query) {
         SQLSyntaxManager syntaxManager = new SQLSyntaxManager();
         syntaxManager.init(dataSource.getSQLDialect(), dataSource.getContainer().getPreferenceStore());
-        return formatSQL(dataSource, syntaxManager, query);
+        return formatSQL(dataSource, syntaxManager, query, null);
     }
 
-    public static String formatSQL(@Nullable DBPDataSource dataSource, @NotNull SQLSyntaxManager syntaxManager, String query) {
+    public static String formatSQL(DBPDataSource dataSource, String query, @Nullable String indent) {
+        SQLSyntaxManager syntaxManager = new SQLSyntaxManager();
+        syntaxManager.init(dataSource.getSQLDialect(), dataSource.getContainer().getPreferenceStore());
+        return formatSQL(dataSource, syntaxManager, query, indent);
+    }
+
+    public static String formatSQL(@Nullable DBPDataSource dataSource, @NotNull SQLSyntaxManager syntaxManager, String query,
+                                   @Nullable String indent) {
```

Comment:
Quite a weird formatting. Also, please use nullability annotations whenever possible.

```suggestion
    @NotNull
    public static String formatSQL(
        @Nullable DBPDataSource dataSource,
        @NotNull SQLSyntaxManager syntaxManager,
        @NotNull String query,
        @Nullable String indent
    ) {
```